app/models/loyalty_program_datum.rb

```diff
@@ -1,9 +1,8 @@
 class LoyaltyProgramDatum < ApplicationRecord
-    has_many :transactions, class_name: "transactions", foreign_key: "Loyalty_program_data_id"
```

Comment:
@mattChrisP This part i am not sure whether to remove

Reply:
can we update on transcation?

Meaning given a transcation, update the 2 loyalty program datum belonging to a user

Reply:
Yeah, we can but we need to set the foreign key from the loyalty program datum as well from what I know. Otherwise, currently, I am not sure how to update that from transaction without having the receiver key as well in the loyalty program datum